src/stretch/agent/debug/dynamem_zmq_client.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# Copyright (c) Hello Robot, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the LICENSE file in the root directory
+# of this source tree.
+#
+# Some code may be adapted from other open-source works with their respective licenses. Original
+# license information maybe found below, if so.
+
+import time
```

Comment:
Standalone apps can go in stretch/app/debug?

Can you add some debugging notes in the doc as to when you should use this?